api/models/Match.js

```diff
@@ -25,7 +25,11 @@ module.exports = {
     },
     endTime: {
       type: 'number',
-      required: true,
+      required: false,
+    },
+    games: {
+      collection: 'Game',
+      via: 'match',
```

Comment:
This lets us query for all the games associated with a specific match, which is used when adding a game to a match to determine if the match should now be over